build-config/resources/Ballerina.toml

```
@@ -0,0 +1,67 @@
+[package]
+org = "ballerina"
+name = "schema_registry"
+version = "@toml.version@"
+authors = ["Ballerina"]
+export=["schema_registry"]
+keywords = ["schema_registry"]
```

Let's use some related keywords here.

```suggestion
keywords = ["confluent", "schema_registry", "avro", "serdes"]
```